.github/workflows/release-please.yaml

```diff
@@ -214,19 +215,20 @@ jobs:
             ./sbom.xml
             ./*.tar.gz
   homebrew:
-   name: Bump homebrew-core formula
-   needs: release-please
-   runs-on: ubuntu-latest
-   # Only run on non-forked flagd releases
-   if: ${{ github.repository_owner == 'open-feature' && needs.release-please.outputs.flagd_tag_name }}
-   steps:
-     - uses: mislav/bump-homebrew-formula-action@v2
-       with:
-         formula-name: flagd
-         tag-name: ${{ needs.release-please.outputs.flagd_tag_name }}
-         download-url: https://github.com/${{ github.repository }}.git
-         commit-message: |
-           {{formulaName}} {{version}}
-           Created by https://github.com/mislav/bump-homebrew-formula-action
-       env:
-         COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
+    name: Bump homebrew-core formula
+    needs: release-please
+    runs-on: ubuntu-latest
+    # Only run on non-forked flagd releases
+    if: ${{ github.repository_owner == 'open-feature' && needs.release-please.outputs.flagd_tag_name }}
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v2
+        with:
+          formula-name: flagd
+          tag-name: ${{ needs.release-please.outputs.flagd_tag_name }}
+          download-url: https://github.com/${{ github.repository }}.git
+          commit-message: |
+            {{formulaName}} ${{ needs.release-please.outputs.flagd_version }}
```

Comment:
According to the documentation of this action [1], commit-message expect the `{{version}}` placeholder. 

And we set this here [2] . Am I missing something 🤔 

Besides, what went wrong here should be the missing new line 

> It's recommended that commit-message has both subject and body, i.e. that it contains a subject line followed by a blank line followed by body text. 


[1] - https://github.com/mislav/bump-homebrew-formula-action#action-inputs
[2] - https://github.com/open-feature/flagd/blob/ef341f3895f9df75bff795e984415c7d7f0dadbd/.github/workflows/release-please.yaml#L161

Reply:
`{{version}}` uses the git tag we're passing in. That is why the PR title was `flagd flagd/v0.5.0` instead of `flagd 0.5.0`. This should address that by not using templating offered by the Homebrew action and instead setting it ourselves.

We explicitly set the flagd version output [here](https://github.com/open-feature/flagd/blob/main/.github/workflows/release-please.yaml#L54). You can see it in action [here](https://github.com/open-feature/flagd/actions/runs/4566549553/jobs/8059489382) under "Dump Release Please Output". The VERSION environment variable you linked is set defined in the `release-go-binaries` job.

Reply:
Thank you. I understand it now :) 

I missed the `$` in the replacement (`${{ needs.release-please.outputs.flagd_version }}`) which is an interpolation.